librarymanagement/src/main/java/org/librarymanagement/controller/admin/AdminAuthController.java

```diff
@@ -0,0 +1,84 @@
+package org.librarymanagement.controller.admin;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpSession;
+import jakarta.validation.Valid;
+import org.librarymanagement.dto.response.LoginResponseDto;
+import org.librarymanagement.dto.request.LoginUserDto;
+import org.librarymanagement.service.LoginService;
+import org.librarymanagement.service.impl.LoginServiceImpl;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminAuthController {
```

Comment:
```suggestion
public class AuthController {
```
ri đã đủ, vì nằm trogn package admin rồi